public/statistiques/agents.php

```diff
@@ -276,6 +269,14 @@
                         $total_absences+=diff_heures($elem['debut'], $elem['fin'], "decimal");
                         $exists_absences=true;
                     }
+
+                    foreach ($agents_list as $elem2) {
+                        if ($elem2['id']==$agent) {	// on créé un tableau avec le nom et le prénom de l'agent.
+                            $agent_tab=array($agent,$elem2['nom'],$elem2['prenom']);
+                            break;
+                        }
+                    }
+
```

Comment:
I moved the creation of the array "agent_tab" outside the condition "if not absent".